source/core/Events.cpp

```diff
@@ -1223,6 +1295,8 @@ DEFINE_VIREO_BEGIN(Events)
                           "i(VarArgRepeat) i(Int32 specIndex)i(StaticTypeAndData)i(BranchTarget))")
 
     DEFINE_VIREO_FUNCTION(_OccurEvent, "p(i(ControlRefNum controlRef) i(UInt32 eSource) i(UInt32 eType))")
+    DEFINE_VIREO_FUNCTION(ConfigureEventSpecJSRef, "p(i(Int32 eStructIndex) i(Int32 eSpecIndex) i(JavaScriptRefNum jsReference))")
```

Comment:
I was thinking about ConfigureEventSpec and RegisterForEvent to make them more generic and possibly useful in other targets, but both functions take a JavaScriptRefNum, so maybe not a good idea. What do you think?

Reply:
Right now it doesn't look we don't allow using just RefNum in these Vireo function definitions, but I can try adding that and see if it's okay taking a generic RefNum

Reply:
The other option there is to add other versions of this definition here that take other RefNum types when we want to support other RefNum types here

Reply:
I think adding other definitions as necessary is preferable; it's likely there would be subtle differences in implementation based on the refnum type anyway.